src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/PlanEntity.kt

```diff
@@ -76,6 +77,14 @@ enum class PublishState {
 interface PlanRepository : JpaRepository<PlanEntity, Long> {
   fun findByUuid(planUuid: UUID): PlanEntity
 
+  @Query(
+    """
+    select p from PlanEntity p
+    where p.uuid = :planUuid
+  """,
+  )
+  fun findPlanByUuId(planUuid: UUID): PlanEntity?
```

Comment:
Can you lowercase the `I` so it's `Uuid`?

Reply:
no